docs/2-fork-accountability.md

```diff
@@ -0,0 +1,161 @@
+# Fork accountability
+
+As we know, the agreement property of the Tendermint consensus algorithm cannot be violated if there are at most *f* faulty processes in the system.
+What happen if more than *f* processes are faulty?
+
+A well-designed consensus protocol should provide some guarantees when this limit is exceeded. The most important guarantee is **fork accountability**, where the processes that caused the consensus to fail can be identified and punished according to the protocol specifications. 
+
+## The problem
+
+We can separate two cases:
+
+- if the number of faulty processes is more than *2f*, we cannot do anything because faulty processes basically control the system and they can do whatever they prefer.
+
+- if the number of faulty processes is less than *2f*, i.e.
+
+      f < num. faulty processes <= 2f
+      
+  Agreement can still be violated but we're in condition to detect the processes that were faulty and punish them, for example by removing them from the validators set. 
+  
+A **fork** happens when two correct validators decide on different blocks in the same height, i.e. there are two commits for different blocks at the same height. 
```

Comment:
Put , (comma) after i.e.

Reply:
Fixed (even for all the other times I use i.e.)